packages/vscode-extension/src/common/Project.ts

```diff
@@ -40,6 +40,14 @@ export type NavigationHistoryItem = {
   id: string;
 };
 
+export type Route = {
```

Comment:
can we rename to `NavigationRoute`. This is a very central interface file that provides definitions for a lot of different functionalities. It would be better for the future if the names we export here are more specific and less generic